tests/development/messages/authentication/test_401_errors.py

```diff
@@ -0,0 +1,47 @@
+import requests

```

Comment:
These tests should not be /v1/messages specific - we already have global 401 errors